### PR TITLE
netbios ports update

### PR DESCRIPTION
**Pull request by abdullahaslam306**

### CODE REVIEW

File: plugins/azure/networksecuritygroups/openNetBIOS.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ module.exports = {
     title: 'Open NetBIOS',
     category: 'Network Security Groups',
     domain: 'Network Access Control',
-    description: 'Determine if UDP port 137 or 138 for NetBIOS is open to the public',
+    description: 'Determine if UDP port 137, 138 or 139 for NetBIOS is open to the public',
```
Comment (alphadev4) [inline plugins/azure/networksecuritygroups/openNetBIOS.js:8]:
```suggestion
    description: 'Determine if  TCP port 139 or UDP ports 137 and 138 for NetBIOS is open to the public',
```

Comment (alphadev4) [reply plugins/azure/networksecuritygroups/openNetBIOS.js:8]: ig 139 is tcp port

[the remark this answers is not in the document]
Comment (alphadev4) [reply plugins/azure/networksecuritygroups/openNetBIOS.js:8]: ig 139 is tcp port

**Review by AkhtarAmir — APPROVED** [review]

File: plugins/azure/networksecuritygroups/openNetBIOS.js [hunk 2]
```diff
@@ -5,10 +5,10 @@ module.exports = {
...
     more_info: 'While some ports such as HTTP and HTTPS are required to be open to the public to function properly, more sensitive services such as NetBIOS should be restricted to known IP addresses.',
     link: 'https://docs.microsoft.com/en-us/azure/virtual-network/manage-network-security-group',
-    recommended_action: 'Restrict UDP ports 137 and 138 to known IP addresses',
+    recommended_action: 'Restrict UDP ports 137, 138 or 139 to known IP addresses',
```
Comment (AkhtarAmir) [inline plugins/azure/networksecuritygroups/openNetBIOS.js:11]:
```suggestion
    recommended_action: 'Restrict UDP ports 137, 138 or 139 and known IP addresses',
```

**Review by AkhtarAmir — APPROVED** [review]

**Pull request merged**